tests/utils.py

```diff
@@ -0,0 +1,38 @@
+import mrob
+import numpy as np
+
+from nptyping import Float, NDArray, Shape
+from src.core import Database
+from src.reduction_methods import ReductionMethod
+
+
+def read_trajectory(path_to_traj: str) -> list[NDArray[Shape["4, 4"], Float]]:
```

Comment:
I suppose all those things should be in common data loader, since we have unified format